scripts/biomedical/NCBI_Assembly/format_NCBI_Chromosome.py

```diff
@@ -70,7 +70,9 @@ def write_genome_assembly(file_output, dict_genome_assembly, genome,
         'Taxid':
             'ncbiTaxonID',
         'BioProject':
```

Comment:
To confirm, are changes in this file intentional?

Reply:
yes